modules/nw-egress-ips-assign.adoc

```diff
@@ -47,5 +47,5 @@ $ oc apply -f <egressips_name>.yaml <1>
 ----
 egressips.k8s.ovn.org/<egressips_name> created
 ----
-
+. Add labels to the namespace which requires egress IP addresses.
```

Comment:
Firstly suggest move this step after   "Optional: Save the `<egressips_name>.yaml` file so that you can make changes later."
Second, suggest give example to label the namespace. Be noted the label should match egressip objects namespaceSelector, that means in this example, the label should be env=qa